actionpack/test/system_testing/screenshot_helper_test.rb

```diff
@@ -0,0 +1,20 @@
+require "abstract_unit"
+
+class ScreenshotHelperTest < ActiveSupport::TestCase
+  def test_driver_support_for_screenshots
+    Rails::SystemTestCase.driver = :rails_selenium_driver
+    assert_equal true, Rails::SystemTestCase.driver.supports_screenshots?
+
+    Rails::SystemTestCase.driver = :rack_test
+    assert_equal false, Rails::SystemTestCase.driver.supports_screenshots?
+
+    Rails::SystemTestCase.driver = :selenium
+    assert_equal true, Rails::SystemTestCase.driver.supports_screenshots?
+
+    Rails::SystemTestCase.driver = :webkit
+    assert_equal true, Rails::SystemTestCase.driver.supports_screenshots?
+
+    Rails::SystemTestCase.driver = :poltergeist
+    assert_equal true, Rails::SystemTestCase.driver.supports_screenshots?
```

Comment:
Why do we want this to return booleans over our generally preferred falsey values?